bundles/org.openhab.binding.remoteopenhab/README.md

```diff
@@ -0,0 +1,64 @@
+# Remote openHAB Binding
+
+The Remote openHAB binding allows to communicate with remote openHAB servers.
+The communication is bidirectional.
+The binding on the local server listens to any item state updates on the remote server and updates accordingly the linked channel on the local server.
+It also transfers any item command from the local server to the remote server.
+
+One first usage is the distribution of your home automation system on a set of openHAB servers.
+
+A second usage is for users having old openHAB v1 bindings running that were not migrated to openHAB v2 or openHAB v3.
+They can keep an openHAB v2 server to run their old openHAB v1 bindings and setup a new openHAB v3 server for everything else.
+The Remote openHAB binding installed on the openHAB v3 server will then allow to use the openHAB v1 bindings through communication with the openHAB v2 server.
+
+A third usage is for users that would like to keep unchanged an existing openHAB v2 server but would like to use the new UI from openHAB v3; they can simply setup a new openHAB v3 server with the Remote openHAB binding linked to their openHAB v2 server.
+
+## Supported Things
+
+There is one unique supported thing : the `server` bridge thing 
+
+## Discovery
+
+All openHAB servers in the local network are automatically discovered (through mDNS) by the binding.
+You will find in the inbox one discovery thing per remote server interface.
+So if your remote server has one IPv4 address and one IPv6 address, you will discover two things in the inbox.
+Just choose one of the two things.
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The thing has the following configuration parameters:
+
+| Parameter | Required | Description                                                                                            |
+|-----------|-------------------------------------------------------------------------------------------------------------------|
+| restUrl   | yes      | The REST URL of the remote openHAB server.                                                             |
```

Comment:
I would find it much more straight forward to simply have the ip address (or hostname) as a configuration parameter - it is imho much more convenient for users when doing textual configuration. Furthermore, that's the standard way we also identify other devices/services on the network.

Reply:
This would require another setting for the port + another to choose between HTTP and HTTPS (when HTTPS will be possible).
And the mDNS announcer is even providing the path ("/rest").
So the REST API URL as configuration setting looks like a good solution, to not say a better solution.
But of course, this can be changed.

Reply:
Ok I am finally convinced, I will update in favor or several settings.

Reply:
Done